package.json

```diff
@@ -47,9 +47,11 @@
     "clean:install": "npm run clean:all && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
     "compile": "lerna run compile",
-    "compile:heroku": "cd packages/terra-clinical-site && npm run compile:heroku",
+    "compile:heroku": "webpack --config webpack.prod.config --p",
```

Comment:
When consuming the new terra-dev-site is there still the need to distinguish between heroku and standard prod when building?

Reply:
good call